packages/core/src/schema/build.ts

```diff
@@ -0,0 +1,61 @@
+import { build } from "esbuild";
```

Comment:
This is more of a code build/loading concern, so I think we should move this file to `build/schema.ts`.

Reply:
Okay, gonna move the config build too